internal/service/streamaccountdetails/data_source_test.go

```diff
@@ -0,0 +1,44 @@
+package streamaccountdetails_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/constant"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+func TestAccStreamAccountDetailsDS_basic(t *testing.T) {
+	var (
+		projectID, _   = acc.ClusterNameExecutionWithRegion(t, constant.UsEast1, false)
```

Comment:
Is this endpoint limited to a set of regions? What is the experience of the end user? 
If its the case want to make sure we have a clear error message of the API side. Potentially we could also clarify this in our docs (I see we currently dont mention anything in API docs/tf docs.